test/e2e/dataplane/service_discovery.go

```diff
@@ -4,34 +4,36 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
-	"time"
 
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	lhframework "github.com/submariner-io/lighthouse/test/e2e/framework"
 	"github.com/submariner-io/shipyard/test/e2e/framework"
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	submarinerIpamGlobalIp = "submariner.io/globalIp"
 )
 
 var _ = Describe("[dataplane] Test Service Discovery Across Clusters", func() {
-	f := framework.NewFramework("dataplane-sd")
+	f := lhframework.NewFramework("dataplane-sd")
 
 	When("a pod tries to resolve a service in a remote cluster", func() {
 		It("should be able to discover the remote service successfully", func() {
 			RunServiceDiscoveryTest(f)
 		})
 	})
-
-	When("a pod tries to resolve a service which is present locally and in a remote cluster", func() {
-		It("should resolve the local service", func() {
-			RunServiceDiscoveryLocalTest(f)
+	/*
+		When("a pod tries to resolve a service which is present locally and in a remote cluster", func() {
+			It("should resolve the local service", func() {
+				RunServiceDiscoveryLocalTest(f)
+			})
```

Comment:
Why is this commented out?

Reply:
Argh. Missed to uncomment after refactoring the dig method.